package.json

```diff
@@ -20,7 +20,8 @@
   },
   "scripts": {
     "build": "node build.js",
-    "build:dev": "RUNTIME_API_HOST='http://localhost:9000' RUNTIME_ENV=dev node build.js",
+    "build:dev": "RUNTIME_API_HOST='https://zerobase-api-stage.herokuapp.com' RUNTIME_ENV=dev node build.js",
```

Comment:
it would be nice to have a script item that runs against localhost for backend developers to test their changes

Reply:
Changed the script to prompt for API host if env var is not provided
![image](https://user-images.githubusercontent.com/22553869/77391747-7b8b3080-6d67-11ea-9f3a-4bfde9b98600.png)